src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/SetAzurePolicyAssignment.cs

```diff
@@ -131,9 +122,9 @@ private JToken GetResource(string resourceId, string apiVersion)
             var policyAssignmentObject = new PolicyAssignment
             {
                 Name = this.Name ?? ResourceIdUtility.GetResourceName(this.Id),
-                Sku = this.Sku != null
-                    ? this.Sku.ToDictionary(addValueLayer: false).ToJson().FromJson<PolicySku>()
-                    : (resource.Sku == null ? new PolicySku { Name = "A0", Tier = "Free" } : resource.Sku.ToJson().FromJson<PolicySku>()),
+                //Sku = this.Sku != null
+                //    ? this.Sku.ToDictionary(addValueLayer: false).ToJson().FromJson<PolicySku>()
+                //    : (resource.Sku == null ? new PolicySku { Name = "A0", Tier = "Free" } : resource.Sku.ToJson().FromJson<PolicySku>()),
                 Properties = new PolicyAssignmentProperties
```

Comment:
Does this blow away parameters and metadata?  Seems broken

Reply:
I verified it is indeed broken in the current cmdlet. It's also the RC of a reported issue. I'll fix it in my next update.

Reply:
Done